booking-up-for-beauty/damoreno/booking_up_for_beauty.go

```diff
@@ -0,0 +1,46 @@
+package booking
+
+import (
+    "fmt"
+    "time"
+)
+
+func parseDate(layout, date string) time.Time {
+    parsedDate, err := time.Parse(layout, date)
+
+    if err != nil {
+        fmt.Println("Oops:", err)
```

Comment:
Here I think you should exit. On error you would continue the program, leading to unknown failures down the line. A shame that the tests do not include failures.